recipes/vcfsim/.ipynb_checkpoints/meta-checkpoint.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "vcfsim" %}
```

Comment:
This file is redundant

Reply:
Thank you, I will fix this right now

Reply:
It doesn't seem to pass the linting test when that line is removed.